### ISSUE TITLE
Gradle Compile Error: Cannot Find v.1.3.11

### ISSUE THREAD

**Issue by CCorrado** [OrangeGangsters/CircularBarPager#11]
Using Android Studio 3.0.1, gradle wrapper v4.4

app/build.gradle Config:
```
    //CircularBarPager
    implementation 'com.github.omadahealth:library:1.3.11@aar'
    //Animations and progress value easing
    implementation 'com.daimajia.easing:library:2.1@aar'
```
project/build.gradle Config:
```
allprojects {
    repositories {
        jcenter()
        maven { url 'http://dl.bintray.com/populov/maven/' }
    }
}
```

Looks like 1.3.11 isn't available yet, getting the following gradle error:

```
Unable to resolve dependency for ':mobile@debug/compileClasspath': Could not resolve com.github.omadahealth:library:1.3.11.
```

Switching to 1.3.10 is fine though. Issue with that is, it still depends on JW's nineoldandroids, which the easing library dropped support for and actually crashes when you try and call...

` circularBar.animateProgress(start, end, len);`

**Comment by 1-riverfish** [comment]
Pay attention to the order of jcenter() and maven { url 'http://dl.bintray.com/populov/maven/' }.
You can refer this website https://stackoverflow.com/questions/29615547/cant-add-viewpagerindicator-to-app-with-gradle
and here is part of my build.gradle file:

allprojects {
    repositories {
        maven { url "http://dl.bintray.com/populov/maven/" }
        jcenter()
        mavenCentral()
    }
}

android {
    compileSdkVersion 28
    defaultConfig {
        applicationId "com.seuchild.smallseedling"
        minSdkVersion 23
        targetSdkVersion 28
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
dependencies {
    implementation fileTree(include: ['*.jar'], dir: 'libs')
    //CircularBarPager
    implementation 'com.github.orangegangsters:library:1.3.4@aar'
    //CirclePageIndicator
    implementation 'fr.avianey.com.viewpagerindicator:library:2.4.1.1@aar'
    //Animations and progress value easing
    implementation 'com.daimajia.easing:library:2.1@aar'
}

Hope you can get something useful.

**Comment by 1-riverfish** [comment]
implementation 'com.github.omadahealth.circularbarpager:library:1.3.11@aar' is avaiable but it still depends on JW's nineoldandroids!!!

**Comment by CCorrado** [comment]
@1-riverfish As a temp fix I ended up compiling the 3 libraries together along with the nineoldandroids library like you say. In any case since March I've actually moved away from using this library because of the dependency problems. The circular animation can be done quite nicely with native tools and proper drawables. Even then, for the ViewPager stuff, Motion Layout is a perfect tool!

https://speakerdeck.com/brittbarak/make-your-app-dance-with-motionlayout Take a look at this presentation for more examples with MotionLayout.
